lib/utils/enumerable_delegation.rb

```diff
@@ -0,0 +1,9 @@
+# encoding: utf-8
+
+module EnumerableDelegation
```

Comment:
This is purpose built for params instead of delegation to some injected receiver, so I wonder if Params should be part of the name? Maybe `EnumerableParamsDelegator`?

Reply:
I didn't like the dependency on the params attribute either, but minimum viable solution and all. I agree that your suggestion is a better name, but I'm definitely more interested in if you have a solution that delegates to an injected receiver. Ideas?